firmware/src/src/AR/oled.cpp

```diff
@@ -42,7 +42,7 @@
   }
 #endif
 
-#define D_S 6 // DIAMOND_SIZE
+#define P_S 6 // POINT_SIZE
```

Comment:
Please use full names (example - POINT_SIZE)

Shouldn't point size be total size (currently size is 2*P_S + 1 - 1 as middle point) - to be considered

Reply:
Full named used

Second comment not addressed, please use #define POINT_HALF_SIZE ((int16_t) POINT_SIZE / 2)

Reply:
Implemented